packages/cli/package-lock.json

```diff
@@ -53,9 +52,9 @@
       }
     },
     "node_modules/@axe-core/webdriverjs": {
-      "version": "4.7.1",
-      "resolved": "https://registry.npmjs.org/@axe-core/webdriverjs/-/webdriverjs-4.7.1.tgz",
-      "integrity": "sha512-Cf3gFDKYkMDo1vO9QQ3uIRP/iyiacxbu6YWFl0LqKG3u4AtUeR64xZ5weexr5xAKixu/sDQmcpjLskkgO9QK9w==",
+      "version": "4.7.3",
```

Comment:
This is an example of the type of stray `package-lock.json` update I mentioned finding concerning in standup earlier - I can live with including it here, just noting it for ease of referencing in retro tomorrow